include/frame.h

```diff
@@ -17,20 +17,27 @@
 #include <vector>
 
 #include "refcount.h"
+#include "streamable_flatbuffer.h"
+#include "../fbs/torchcraft_generated.h"
 
 #ifdef _MSC_VER
 typedef unsigned int uint32_t;
 typedef unsigned short uint16_t;
 typedef int int32_t;
 #endif
 
-// TODO Check types !
-
 namespace torchcraft {
 namespace replayer {
+
+class Frame;
+class FrameDiff;
+std::ostream& operator<<(std::ostream& out, const Frame& o);
+std::istream& operator>>(std::istream& in, Frame& o);
+std::ostream& operator<<(std::ostream& out, const FrameDiff& o);
+std::istream& operator>>(std::istream& in, FrameDiff& o);
```

Comment:
Pulled the stream operators up to one place for my own sanity. Will probably put them back next to their respective class definitions. Ultimately I think we want to avoid using these and go FlatBuffers all the way down because it potentially eliminates copy steps, but for now the Lua/Python clients will still be using these.